sourcemash/frontend/static/js/views/browse_view.js

```diff
@@ -0,0 +1,66 @@
+Sourcemash.Views.BrowseView = Backbone.View.extend({
+  template: JST['browse'],
+
+  initialize: function (options) {
+    var ExtendedTypeahead = Backbone.Typeahead.extend({
+      template: JST['new-feed-form'],
+    });
+
+    this.typeahead = new ExtendedTypeahead({collection: this.collection, key: 'title'});
```

Comment:
Do we still have the new_feed_form on the users' feeds page? If so, should we remove it?